packages/web/config/generate-chain-infos/source-chain-infos.ts

```diff
@@ -3689,6 +3689,42 @@ const mainnetChainInfos: SimplifiedChainInfo[] = [
     features: ["ibc-transfer", "ibc-go"],
     explorerUrlToTx: "https://www.mintscan.io/neutron/txs/{txHash}",
   },
+  {
+    rpc: "https://rpc-composable-ia.cosmosia.notional.ventures",
+    rest: "https://api-composable-ia.cosmosia.notional.ventures",
+    chainId: "centauri-1",
+    chainName: "Centauri",
+    bip44: {
+      coinType: 118,
+    },
+    bech32Config: Bech32Address.defaultBech32Config("centauri"),
+    currencies: [
+      {
+        coinDenom: "PICA",
+        coinMinimalDenom: "ppica",
+        coinDecimals: 12,
+        coinGeckoId: "pool:ppica",
+        coinImageUrl: "/tokens/composable.png",
+        isStakeCurrency: true,
```

Comment:
I'm getting the error 'stakeCurrency" is required, even though there is a stakeCurrency defined....

Reply:
This should be fixed by using #1735 assets generation.

Reply:
Yes, it does--thanks!